### ISSUE TITLE
Spacecraft GLTF model not loading without any errors

### ISSUE THREAD

**Issue by greenmughal** [bevyengine/bevy#2202]
## Bevy version
e3435e5144b79908d22eabcefb7bee92d5f520ba

## Operating system & version

Ubuntu20
Intel UH

## What I did
I downloaded this free model (the blender version) and have exported to GLTF in all 3 formats without luck to see anything or error
https://www.cgtrader.com/free-3d-models/space/spaceship/star-sparrow-modular-spaceship

All I see is a black screen.

Thanks

**Comment by mockersf** [comment]
could you share the gltf model you tried using, and the code you used?

**Comment by greenmughal** [comment]
Unzip this GLB format
[StarSparrow.zip](https://github.com/bevyengine/bevy/files/6498377/StarSparrow.zip)

Unzip this Embeded GLTF format
[StarSparrow.zip](https://github.com/bevyengine/bevy/files/6498381/StarSparrow.zip)

Unzip this normal GLTF
[StarSparrow.zip](https://github.com/bevyengine/bevy/files/6498386/StarSparrow.zip)

You can view them in any GLTF2 viewer (exported  from blender) to view their hierarchy.

I used exactly the same code as example load_gltf

**Comment by mockersf** [comment]
Loading the glb file, I get the following error:
```
thread 'Compute Task Pool (2)' panicked at 'Attribute Vertex_Tangent is required by shader, but not supplied by mesh. Either remove the attribute from the shader or supply the attribute (Vertex_Tangent) to the mesh.', crates/bevy_render/src/pipeline/pipeline_compiler.rs:231:17
```

This should be fixed by #1795, or you can add the tangents yourself to the model

**Comment by greenmughal** [comment]
Thanks how can I debug such issues? I dont see any log of it.

**Comment by rparrett** [comment]
I can confirm that this was fixed by #1795 which is included in Bevy 0.8+
